source/frontend/src/features/mapSideBar/consolidation/AddConsolidationView.tsx

```diff
@@ -0,0 +1,219 @@
+import { exists } from '@/utils';
+
+import MapSideBarLayout from '../layout/MapSideBarLayout';
+import { AddressForm, PropertyForm } from '../shared/models';
+import SelectedOperationProperty from '../shared/operations/SelectedOperationProperty';
+import SelectedOperationPropertyHeader from '../shared/operations/SelectedOperationPropertyHeader';
+import SidebarFooter from '../shared/SidebarFooter';
+import AddConsolidationMarkerSynchronizer from './AddConsolidationMarkerSynchronizer';
+import { ConsolidationFormModel } from './AddConsolidationModel';
+
+export const AddConsolidationYupSchema = Yup.object().shape({
+  pid: Yup.string()
+    .nullable()
+    .matches(/^\d{0,3}-\d{3}-\d{3}$|^\d{0,9}$/, 'Invalid PID'),
+  sourceProperties: Yup.array().test({
+    message: 'You must select at least two parent properties',
+    test: arr => !!arr?.length && arr.length >= 2,
+  }),
+  destinationProperty: Yup.object().nullable().required('You must select a child property'),
+});
+
+export interface IAddConsolidationViewProps {
+  formikRef: React.RefObject<FormikProps<ConsolidationFormModel>>;
+  consolidationInitialValues: ConsolidationFormModel;
+  loading: boolean;
+  displayFormInvalid: boolean;
+  onSubmit: (
+    values: ConsolidationFormModel,
+    formikHelpers: FormikHelpers<ConsolidationFormModel>,
+  ) => void | Promise<any>;
+  onCancel: () => void;
+  onSave: () => void;
+  getPrimaryAddressByPid: (pid: string) => Promise<AddressForm | undefined>;
+  MapSelectorComponent: React.FunctionComponent<IMapSelectorContainerProps>;
+  PropertySelectorPidSearchComponent: React.FunctionComponent<
+    React.PropsWithChildren<PropertySelectorPidSearchContainerProps>
+  >;
+}
+
+const AddConsolidationView: React.FunctionComponent<
+  React.PropsWithChildren<IAddConsolidationViewProps>
+> = ({
+  formikRef,
+  consolidationInitialValues,
+  loading,
+  displayFormInvalid,
+  onSubmit,
+  onSave,
+  onCancel,
+  getPrimaryAddressByPid,
+  MapSelectorComponent,
+  PropertySelectorPidSearchComponent,
+}) => {
+  return (
+    <MapSideBarLayout
+      showCloseButton
+      title="Create a Consolidation"
+      icon={<StyledSubdivideConsolidateIcon />}
+      onClose={onCancel}
+      footer={
+        <SidebarFooter
+          isOkDisabled={loading}
+          onSave={onSave}
+          onCancel={onCancel}
+          displayRequiredFieldError={displayFormInvalid}
+          saveButtonLabel="Create Consolidation"
+        />
+      }
+    >
+      <StyledFormWrapper>
+        <LoadingBackdrop show={loading} />
+        <Formik<ConsolidationFormModel>
+          onSubmit={onSubmit}
+          initialValues={consolidationInitialValues}
+          innerRef={formikRef}
+          validationSchema={AddConsolidationYupSchema}
+        >
+          {({ values, setFieldValue, errors }) => (
+            <Form>
+              <Section>
+                <H2>
+                  Properties in Consolidation &nbsp;
```

Comment:
nit: use CSS instead of '& nbsp;" to fix presentation.

Reply:
Not sure where these are coming from. I guess that some of the linting rules are not working coherently between the different tools we are using.